AudioPlayerRecorderLibrary/src/main/java/com/heavyplayer/audioplayerrecorder/service/manager/ServiceManager.java

```diff
@@ -37,9 +40,16 @@ final public void onStop() {
     protected void onActivateService() {
         bindService();
 
-        // Ensure service keeps running until we explicitly stop it,
-        // which is always except when configuration changes occur.
-        startService();
+        ActivityManager activityManager = (ActivityManager) mActivity.getSystemService(Context.ACTIVITY_SERVICE);
+        List<ActivityManager.RunningAppProcessInfo> runningAppProcesses = activityManager.getRunningAppProcesses();
+        if (runningAppProcesses != null) {
+            int importance = runningAppProcesses.get(0).importance;
```

Comment:
Should we find our process or are we certain it's always the first one?

Reply:
It should be the first one as that's the foreground one. I've added that in the failure callback though so we can log it, just in case there's something else amiss.